templates/log-book-report/templates/main.tpl

```diff
@@ -0,0 +1,193 @@
+{{define "received-header"}}
+    {{range $item := (getSlice "#,Manufacturer,Model,VIN,DateReceived,Source")}}
```

Comment:
Maybe create a variable for `#,Manufacturer,Model,VIN,DateReceived,Source` as you use it here and in the next block as well.